context/languagesContext.js

```diff
@@ -0,0 +1,18 @@
+import { createContext, useEffect, useState } from "react";
+import languageMock from "../mock/languages";
+
+export const LanguagesContext = createContext();
+
+export default function LanguagesContextProvider({ children }) {
+  const [languages, setLanguages] = useState(languageMock);
+
+  useEffect(() => {
+    console.log(languages);
```

Comment:
does the console.log serve any purpose in the useffect?

Reply:
Thank you, just forgot it - no purpose at all :)